app/locations/views/locations.njk

```diff
@@ -29,13 +29,15 @@
         <ul class="govuk-list govuk-!-font-size-24 govuk-!-font-weight-bold">
           {% for location in locations %}
             <li>
-              <a href="/locations/{{ location.id }}">
                 {% if location.type == "regions" %}
-                  {{ location.key }} {{ location.name }}
+                  <a href="/locations/{{ location.id }}?isRegion=true">
```

Comment:
I wonder if it might be nicer to separate the two features. Does this approach mean that somebody who sees regions cannot access a single location anymore?

Reply:
That is something I did wonder about. Does PMU ever need to access a single location?

Reply:
> Does this approach mean that somebody who sees regions cannot access a single location anymore?
This is addressed in a future ticket: P4-1607